app/models/manageiq/providers/openstack/inventory/collector/cloud_manager.rb

```diff
@@ -55,6 +55,22 @@ def host_aggregates
     @host_aggregates = safe_list { compute_service.aggregates.all }
   end
 
+  def placement_group_by_vm_id
+    @placement_group_by_vm_id ||= placement_groups.each_with_object({}) { |sg, result| sg.members.each { |vm_id| result[vm_id] = sg } }
+  end
+
+  def placement_groups
+    return @placement_groups if @placement_group && @placement_groups.any?
+
+    @placement_groups = compute_service.handled_list(:server_groups, {}, openstack_admin?)
+  end
+
+  def server_group_by_vm_id
```

Comment:
Is there a difference between placement group and server group?

Reply:
no difference.

Reply:
It is common practice to have the nomenclature in the Collector match the native provider, since the Parser is responsible for the translation between provider-native and MIQ-generic